channels.go

```diff
@@ -39,14 +39,12 @@ type Channel struct {
 	Creator            string         `json:"creator"`
 	IsArchived         bool           `json:"is_archived"`
 	IsGeneral          bool           `json:"is_general"`
-	IsGroup            bool           `json:"is_group"`
-	IsStarred          bool           `json:"is_starred"`
```

Comment:
I just connected to the websocket api and the first info payload return contains channels with the is_starred property.